vscode-extension/src/extension.ts

```diff
@@ -42,6 +123,242 @@ export function deactivate() {
   console.log("Deactivated AIConfig extension");
 }
 
+/**
+ * Creates a new AIConfig file in the editor.
+ */
+async function createNewAIConfig(
+  context: vscode.ExtensionContext,
+  aiconfigEditorManager: AIConfigEditorManager,
+  mode: "json" | "yaml" = "json"
+) {
+  const workspaceUri = vscode.workspace.workspaceFolders
+    ? vscode.workspace.workspaceFolders[0].uri
+    : null;
+  const untitledUri = workspaceUri
+    ? workspaceUri.with({
+        scheme: "untitled",
+        path: `${workspaceUri.path}/untitled.aiconfig.${mode}`,
+      })
+    : vscode.Uri.parse(`untitled:untitled.aiconfig.${mode}`);
+
+  // Specify the initial content here
+  const newAIConfigJSON = vscode.Uri.joinPath(
+    context.extensionUri,
+    "static",
+    "untitled.aiconfig.json"
+  );
+
+  const newAIConfigYAML = vscode.Uri.joinPath(
+    context.extensionUri,
+    "static",
+    "untitled.aiconfig.yaml"
+  );
+
+  const fileContentPath = mode === "json" ? newAIConfigJSON : newAIConfigYAML;
+
+  const fileContentBuffer = await vscode.workspace.fs.readFile(fileContentPath);
+  const initialContent = fileContentBuffer.toString();
```

Comment:
Do we just let the server initialize with empty config json?

Reply:
Ah, wait, I realize now that the above is loading default contents from untitled.aiconfig files, right? Could we rename those files to make it clearer they're the default templates for new config files?